Checkout (#43)
data-lessons/instructor-training: _episodes/20-checkout.md

- "To certify, you must contribute to a lesson, take part in a discussion, and do a teaching demo within 90 days of your training event."
---

In this short episode, we will take a moment to review the actions you will
need to take after this training to become a fully certified Carpentries Instructor.  

## Instructor Checkout

After completing the Instructor Training workshop, there are three final steps to complete before qualifying as an Instructor. 
Briefly, the three steps are:

1.  Make (and send us a link to) a small contribution to a lesson or glossary.
2.  Take part in an online community discussion session.

> 3.  Take part in an online teaching demonstration session.

Author: I think this needs a step 4 (demo)
```suggestion
3.  Take part in an online teaching demonstration session.
4.  This is my suggestion
```

All trainees have 3 months (90 days) from the end date of your training to complete checkout. If you need more time,
3-month extensions may be requested by emailing [checkout@carpentries.org](mailto: checkout@carpentries.org). 
Extensions may be granted for any reason up to 1 year from your training date.

> ## Be The Expert: Checkout Q & A
>
> In small groups, read and discuss one of the three checkout procedures described on this page. (https://carpentries.github.io/instructor-training/20-checkout/index.html if you are viewing this text elsewhere)

> > Make notes in the Etherpad and when you are done, report back to the full group about the requirements for that stage of the process.

Reviewer: Make notes about what? Can this activity be clearer?

Author: I moved the questions at the end of the prompt to bullets following "make notes" -- maybe this helps?

> What points do you think it is most important or helpful for people to remember? What questions do you still have about the checkout process?
>
> This exercise should take about 5 minutes.
{: .challenge}

## 1. Lesson Contributions

### Keep It Simple!

> **The purpose of the lesson contribution is to demonstrate that you are prepared to draw upon your experience teaching our lessons to

Reviewer: ```suggestion
**The purpose of the lesson contribution is to demonstrate that you are prepared to draw upon your experience teaching Carpentries lessons to
```

contribute towards their maintenance and improvement in the future.** For this reason, we suggest starting small! Consider commenting

> on an existing issue, fixing a typo if you find one, or contributing a single term to our multilingual glossary. If you would like

Reviewer: People unfamiliar with github won't know what an issue is. Can we put a link on the word issue that either takes us to a definition of a github issue or to an example?

to help us out with something more substantial, consider selecting an item from our [Help Wanted page][help-wanted]. 

### Eligible Repositories
Contributions to repositories that are not managed by The Carpentries, including forks of Carpentries repositories, are **not** eligible for
checkout contributions. This is because The Carpentries Code of Conduct does not apply outside of Carpentries spaces. 

We recommend that contributions be directed to:

> * any [Software Carpentry]({{ site.swc_site }}/lessons/), [Data Carpentry]({{ site.dc_site }}/lessons/), or [Library Carpentry]({{ site.lc_site }}/lessons/) lesson

Reviewer: I have been told that lesson programs should be listed in alphabetical order

Author: ```suggestion
* any [Data Carpentry]({{ site.dc_site }}/lessons/), [Library Carpentry]({{ site.lc_site }}/lessons/), or [Software Carpentry]({{ site.swc_site }}/lessons/) lesson
```

* any [community developed lesson][carpentries-incubator]
* our open source, multilingual glossary of data science terms, [Glosario][glosario-github]

> Wiht the exception of Glossario, **please be cautious about proposing new content or exercises**. Most of our lessons are full to bursting! 

Reviewer: ```suggestion
With the exception of Glossario, **please be cautious about proposing new content or exercises**. Most of our lessons are full to bursting! 
```

The greater challenge, in most cases, is identifying where and how to cut. 

### Contributing on GitHub

> Each lesson repository has a document, named `CONTRIBUTING.md`, that outlines contribution guidelines. The one for this repository

Reviewer: Similar to my comment above about people unfamiliar with github not knowing what an issue is, they may not know what a repo is. Could we create a call out or link to a page with some github basics and/or definitions?

Reviewer: ah, I see that we already have exactly this in the next section. Maybe we can point to https://github.com/dmgt/swc_github_flow/blob/master/for_novice_contributors.md earlier?

> please [let us know][contact-page].
{: .callout}

### What Comes Next?
If you submit your own contribution, be sure to send an email with a link to [checkout@carpentries.org](mailto: checkout@carpentries.org). 
Pay attention to GitHub notifications in case the Maintainers or others have follow-up questions or 
requests. However, also keep in mind that Carpentries lesson Maintainers, like Instructors, are mostly volunteers! Some repositories are vigorously
maintained; others may have slower response times. Do not take it personally if your contribution does not get a prompt response.
**Your contribution does not need to be reviewed or accepted in order for you to become an Instructor**.

## 2. Community Discussion

> The purpose of having you attend a Community Discussion meeting is to introduce you to other members of our global community

Reviewer: You call it a session above rather than a meeting

Author: ```suggestion
The purpose of having you attend a Community Discussion is to introduce you to other members of our global community
```

and to demonstrate an ongoing resource for support and advice as you develop your teaching practice. 

Community Discussion meetings occur several times a week and may take different forms. The most common type of Community Discussion 
meeting is designed to support Instructors in preparing to teach an upcoming workshop and to provide a forum for sharing
lessons learned by those who have recently taught. If there are no workshops to discuss on the agenda, then the facilitator will 
lead a conversation about teaching and preparing to teach, engaging the interests of participants. These meetings are most often 
led by a member of the Instructor Development committee or The Carpentries Core Team.

In some cases, a Community Discussion meeting may be used for regional community-building, to invite the community to connect
with representatives from specific Carpentries committees or subcommunities, or to address a theme of special interest to our
community.

> When signing up on the [Etherpad][discussion], be sure to examine the description of your event carefully. Also, **take care to double check

Reviewer: ```suggestion
When signing up on the [Community Discussion Etherpad][discussion], be sure to examine the description of your event carefully. Also, **take care to double check
```

the time zone**, and mark your calendar after signing up! At the time of the event, the Zoom link may be found at the top
of the Etherpad.

> ## Carpentries Tools: Etherpad
>

> > The Etherpad is a widely used tool in Carpentries workshops and many other activities in the community.

Reviewer: ```suggestion
> The Etherpad is a collaborative note taking tool that is widely used in Carpentries workshops and many other activities in the community.
```

Reviewer: By this point in the document, we have already talked about etherpads several times. Can we move this to a call out the first time we talk about it or link to this section the first time we talk about it?

> We also use Etherpads for sign-ups and notes on recurring events, like community discussions and teaching demonstrations.
> Most Etherpads for recurring events are linked from our ["pad of pads"][pad-of-pads].
> This is a great one to bookmark and use as a reference throughout your checkout process and as a newly minted member
> of The Carpentries community.
>
{: .callout}

> ### What Should I Say?

Reviewer: ```suggestion
### What Should I Say at a Community Discussion Session?
```
Alternatively ### What Should I Expect at a Community Discussion Session?

> Joining a meeting to "discuss" training with strangers can feel a little intimidating! If you have enjoyed meeting the people at your Instructor Training 

Reviewer: The word training is a little confusing here. Do you mean the training the trainee has received or do you mean the teaching they will go on to do?

workshop (we hear this a lot!) you will probably enjoy meeting people at your discussion session, too. 

> Being prepared to participate does not mean you need to do any homework in advance! However, it can help to put a little thought into 

Reviewer: I love this addition! I was so nervous and definitely did prep ahead of my first discussion!

what questions you have about training. In case you need a little inspiration, here are a few themes to consider:
* distribution of teaching among the instructional team
* distribution of organisational responsibilities among the instructional team
* adaptations or modifications to curricular materials
* timing and pacing
* sticking points or challenges

> * content questions or concerns 

Reviewer: ```suggestion
* questions or concerns about the curricula  
```

* broader questions about the value or experience of teaching Carpentries workshops

> ### Not Just for Checkout!

Reviewer: ```suggestion
### Community Discussions are Not Just for Checkout!
```

Community Discussion meetings are an excellent opportunity to

> get help as you prepare to teach and to engage in vital reflection on your practice while connecting with a community of like-

Reviewer: ```suggestion
get help as you prepare to teach and to engage in vital reflection on your teaching practice while connecting with a community of like-
```

minded Instructors. You are invited to attend these at any time! If you think you might enjoy hosting them, or would like to help develop 

> other opportunities for Instructors, consider joining the Instructor Development Committee. 

Reviewer: ```suggestion
other opportunities for Instructors, consider joining the [Instructor Development Committee](https://carpentries.org/inst-dev/). 
```

Author: This page is out of date. I have added it to my [request for updates to the Community page](https://github.com/carpentries/carpentries.org/issues/997), and will add this link if it is updated.

> ## Ongoing support
>

> > Instructors are highly encouraged to participate in discussions before and after each of their workshops,

Reviewer: ```suggestion
> Instructors are highly encouraged to participate in Community Discussions before and after each workshop they teach,
```

> > so that they can continue to learn from each other and advance their teaching skills. For even more support, or 

Reviewer: The phrase "For even more support" feels weird to me and I can't quite articulate why, so feel free to ignore this. But, maybe we could consider starting the sentence at "To work more intensively..."

preparations when the date of that workshop approaches. Finally, it is an opportunity to receive **feedback from a Carpentries 
Instructor Trainer** (like your Trainers at today's event), who will help make sure you are prepared for a strong start.

### Not a High Stakes Test
It can be nerve-wracking to perform in front of others! This is true when you teach, too. As much as possible, try to relax and
enjoy this friendly opportunity for feedback and connection with others in our community. Remember to embrace your mistakes!

In some cases, an Instructor Trainer may ask you to repeat your teaching demonstration another time prior to certification. If this
happens, they will be very clear about exactly what they would like you to change on your second appearance, and why. These requests
are made with your interests in mind -- we want you to be prepared for success when you set out to teach! 

Trainers draw from their own experience and training when they evaluate teaching demonstrations, but we do provide a 
[rubric][demos-rubric] for their reference. This can be a helpful resource as you prepare. 

> ### 1 Lesson = Many Episodes: preparing to teach a lesson.

Reviewer: ```suggestion
### 1 Lesson = Several Episodes: preparing to teach a lesson.
```
Many sounds scarier than several

> ### 1 Lesson = Many Episodes: preparing to teach a lesson.

Reviewer: This discussion of lesson vs. episode is much better, but still feels a bit confusion, especially when we are talking about starting points and the lesson table.

Author: I'm going to leave this for now based on rumors that the vocabulary may be changing...

For your teaching demonstration, you will prepare to teach **a complete lesson** from one of the

> Carpentries lesson programs. Please refer to the lists of [Software Carpentry lessons]({{ site.swc_site }}/lessons/), [Data Carpentry lessons]({{ site.dc_site }}/lessons/), and [Library Carpentry lessons]({{ site.lc_site }}/lessons/) on the websites. 

Reviewer: Again, I think these should be listed alphabetically

Author: ```suggestion
Carpentries lesson programs. Please refer to the lists of [Data Carpentry lessons]({{ site.dc_site }}/lessons/), [Library Carpentry lessons]({{ site.lc_site }}/lessons/), and [Software Carpentry lessons]({{ site.swc_site }}/lessons/) on the websites. 
```

> A "lesson" corresponds to a single line in the lesson table and a single repository in GitHub.

Reviewer: "a single line in the lesson table" seems like it could be confusing

Author: So far relying on the table structure to define these terms has led to substantial improvements in trainees understanding this difficult language, so I am hesitant to change it. If you have a specific suggestion that you feel would make it more clear without changing the table reference, feel free to give it a shot.

(An example of a lesson: [R for Reproducible Scientific Analysis][r-gapminder].
Some lessons have supplementary modules.
You do not need to be prepared to teach the supplementary modules for your teaching demonstration.

> **The Trainer hosting the session session may choose any episode of the lesson** from this [list of suggested episodes][start-points] for you to teach. (An 

Reviewer: ```suggestion
**The Trainer hosting the teaching demonstration session may choose any episode of the lesson** from this [list of suggested episodes][start-points] for you to teach. (An 
```

example for an episode could be: [Data Structures][r-gapminder-episode]). Because we are particularly interested in your use of participatory live coding (or 

> participatory work with spreadsheets etc.), you will not be asked to teach episodes that are entirely expository or that have dependencies or 

Reviewer: is there a word we could use that is clearer and more "plain language" than expository? 

Also, it may not be clear to everyone what dependencies means in this context

Author: ```suggestion
participatory work with spreadsheets etc.), you will not be asked to teach episodes that lack participatory content or that require files or other output from previous episodes.
```

what questions you have about training. In case you need a little inspiration, here are a few themes to consider:
* distribution of teaching among the instructional team
* distribution of organisational responsibilities among the instructional team
* adaptations or modifications to curricular materials
* timing and pacing
* sticking points or challenges
* questions or concerns about the curricula  
* broader questions about the value or experience of teaching Carpentries workshops

### Community Discussions are Not Just for Checkout!
Community Discussion meetings are an excellent opportunity to
get help as you prepare to teach and to engage in vital reflection on your teaching practice while connecting with a community of like-
minded Instructors. You are invited to attend these at any time! If you think you might enjoy hosting them, or would like to help develop 
other opportunities for Instructors, consider joining the Instructor Development Committee. 

> ## Ongoing support
>
> Instructors are highly encouraged to participate in Community Discussions before and after each workshop they teach,

> > so that they can continue to learn from each other and advance their teaching skills. For even more support, or 
> > to work more intensively on specific aspects of your teaching practice, consider

Author: ```suggestion
> so that they can continue to learn from each other and advance their teaching skills. To work more intensively on specific aspects of your teaching practice, consider
```
